### DISCUSSION TITLE
Works locally but authorize not called in production (Docker running on AWS ECS)

### DISCUSSION THREAD

**Discussion by lominming** [nextauthjs/next-auth#8055] [Help]
### Question 💬

**Setup:**
- "@auth/core": "^0.9.0"
- "@auth/solid-start": "^0.1.1"
- "solid-start": "^0.2.26"
- "solid-js": "^1.7.2"

**Environment:**
- Docker in AWS ECS

**Provider:**
- Credentials (email + password)

**Env**:
- AUTH_TRUST_HOST=true
- AUTH_SECRET=xyz
- AUTH_URL=https://url
- NEXTAUTH_URL=https://url
- AUTH_URL_INTERNAL=https://url
- NEXTAUTH_URL_INTERNAL=https://url

**Problem:**
- Everything works fine locally, but in production, it never call `authorize` (the function we set up in `SolidAuthConfig`)
- It felt like a configuration issue, but have already setup all the necessary variables
- Checked the source code for @auth/solid-start but it looked very straight-forward

Unclear how best to trace and debug what is going on

### How to reproduce ☕️

```
export const authOpts: SolidAuthConfig = {
  trustHost: true,
  secret: SOLID_AUTH_SECRET, 
  debug: true,
  providers: [
    CredentialsProvider({
      id: 'credentials',
      name: 'Password',
      credentials: {
        email: { label: 'Email', type: 'text', placeholder: 'example@example.com' },
        password: { label: 'Password', type: 'password', maxLength: 50 },
      },
      authorize: async (credentials): Promise<User | null> => {
        console.log('authorize called'); // THIS IS NOT CALLED ON PRODUCTION
        ...
      },
    }),
  ],
```

### Contributing 🙌🏽

Yes, I am willing to help answer this question in a PR

**Comment by zimonitrome** [comment]
Did you solve this?

I feel like I have a similar issue: https://github.com/nextauthjs/next-auth/discussions/8153
